src/Analyzer/RstAnalyzer.php

```diff
@@ -107,15 +97,10 @@ public function analyze(\SplFileInfo $file, array $rules): array
                     continue;
                 }
 
-                $violationMessage = $rule->check($lines, $no);
+                $violation = $rule->check($lines, $no, (string) $file->getRealPath());
 
-                if (null !== $violationMessage) {
-                    $violations[] = Violation::from(
-                        $violationMessage,
-                        (string) $file->getRealPath(),
-                        $no + 1,
```

Comment:
Before we had here `$no + 1`, but afterwards, all LineContentRules are using:
```php
           return Violation::from(
                 'a message'
                 $filename,
                 1,
                 ''
             );
```

And Line number 1, or am I missing sth @AlessandroMinoccheri 🤔

Reply:
Also the raw line is missing 🤔

Reply:
Spotted while working on https://github.com/OskarStark/doctor-rst/pull/1276

Reply:
It would be awesome if you can restore the number and the raw line for the violations in a new PR 🙏